publish.rb

```diff
@@ -31,7 +31,7 @@
     FileUtils.cp_r "_site/.", tmp
     Dir.chdir tmp
     system "git init"
-
+    
```

Comment:
I had some test code here. It's a good practice as far as I am concerned to have whitespace on blank lines between code anyway, so I kept it.